### PR TITLE
Remove some unused frameworks

### PR DESCRIPTION
**Pull request by darinwf**
I think these became unnecessary when we removed the web3 code.

### CODE REVIEW

**Comment by yozuysal** [comment]
Kanna is used in ShareTo for parsing html head for og tags.

**Comment by darinwf** [comment]
> Kanna is used in ShareTo for parsing html head for og tags.

Thanks for catching that! I forgot to grep through that directory, and sure enough CI caught it as well.

File: Client.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -5200,7 +5196,6 @@
 			);
 			name = SharedTests;
 			packageProductDependencies = (
-				E96D956E2832FDF100DD5BB3 /* Yams */,
```
Comment (EdwardJTL) [inline Client.xcodeproj/project.pbxproj:5203]: yams is used in testing

File: Client.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -9332,14 +9321,6 @@
 /* End XCConfigurationList section */
 
 /* Begin XCRemoteSwiftPackageReference section */
-		1C45B366283C4F36008A1401 /* XCRemoteSwiftPackageReference "CodeScanner" */ = {
```
Comment (EdwardJTL) [inline Client.xcodeproj/project.pbxproj:9335]: i thought code scanner was used for the QR code sign in flow. @yasminyin96 is that correct?

Comment (darinwf) [reply Client.xcodeproj/project.pbxproj:9335]: It is, but it is included twice. This is just removing the second instance.

**Review by jedireza — APPROVED** [review]
Build is passing. That should be the bar I think. 😉

**Review by EdwardJTL — APPROVED** [review]

**Pull request merged**